blackboxopt/base.py

```diff
@@ -104,14 +116,15 @@ def get_evaluation_specification(self) -> EvaluationSpecification:
         """
 
     @abc.abstractmethod
-    def report_evaluation(self, evaluation: Evaluation) -> None:
-        """Report an evaluated evaluation specification.
+    def report(self, evaluations: Union[Evaluation, Iterable[Evaluation]]) -> None:
```

Comment:
You mentioned having seen that paradigm with pandas. Do they use singular or plural names for the combined arguments that allow both single and multiples?

Reply:
Plurals, I would do it the same (It's more intuitions that passing one evaluation is okay if many are also possible, than the other way round.)